client/packages/system/src/Name/SupplierDetailView/PurchaseOrder.tsx

```diff
@@ -1,57 +1,73 @@
 import React, { ReactElement } from 'react';
 import {
   Box,
+  ColumnDefinition,
+  ColumnFormat,
   createTableStore,
   DataTable,
   NothingHere,
   TableProvider,
   useColumns,
   useTranslation,
 } from '@openmsupply-client/common';
-
-// TODO:
-// This is still to be connected to the backend
-// Column definitions and data are placeholders
-// Labels are placeholders and should be replaced with actual translations
+import { PurchaseOrdersFragment } from '../apiModern/operations.generated';
+import { usePurchaseOrders } from '../apiModern';
 
 export const PurchaseOrder = (): ReactElement => {
   const t = useTranslation();
+  const { data } = usePurchaseOrders();
 
-  const columns = useColumns(
-    [
-      {
-        key: 'orderNumber',
-        label: 'Order Number',
-      },
-      {
-        key: 'status',
-        label: 'Status',
-      },
-      {
-        key: 'date',
-        label: 'Date',
-      },
-      {
-        key: 'lock',
-        label: 'Lock',
-      },
-      {
-        key: 'targetDays',
-        label: 'Target Days',
-      },
-      {
-        key: 'lines',
-        label: 'Lines',
-      },
-      {
-        key: 'comment',
-        label: 'Comment',
-      },
-    ],
+  const columnDefinitions: ColumnDefinition<PurchaseOrdersFragment>[] = [
     {
-      sortBy: { key: 'orderDate', direction: 'desc' },
-    }
-  );
+      key: 'supplier',
+      label: 'label.supplier',
+      accessor: ({ rowData }) => rowData.supplier?.name ?? '',
+    },
+    {
+      key: 'orderNumber',
+      label: 'label.number',
+      accessor: ({ rowData }) => rowData.number ?? '',
+    },
+    {
+      key: 'createdDatetime',
+      label: 'label.created',
+      format: ColumnFormat.Date,
+      accessor: ({ rowData }) => rowData.createdDatetime ?? '',
+    },
+    {
+      key: 'confirmationDate',
+      label: 'label.confirmed',
+      format: ColumnFormat.Date,
+      accessor: ({ rowData }) => rowData.confirmedDatetime ?? '',
+    },
+    {
+      key: 'status',
+      label: 'label.status',
+      accessor: ({ rowData }) => rowData.status ?? '',
```

Comment:
I think we need to translate these statuses to different languages.

Here's what we do for an inbound shipment...
https://github.com/msupply-foundation/open-msupply/blob/2d332a4730ec8ffa227c36426999e459e2456dbb/client/packages/invoices/src/InboundShipment/ListView/ListView.tsx#L79